ext/js/app/frontend.js

```diff
@@ -119,6 +119,7 @@ export class Frontend {
 
         this._hotkeyHandler.registerActions([
             ['scanSelectedText', this._onActionScanSelectedText.bind(this)],
+            ['scanTextAtSelection', this._onActionScanTextAtSelection.bind(this)],
```

Comment:
Honestly if renaming is possible I would rather name the current `scanSelectedText` to `scanTextGreedy` and repurpose `scanSelectedText` for this new functionality. I'm not sure about the naming of "lazy" tbh

Reply:
Thought about naming them like that but I feel like "greedy" and "lazy" aren't as well known terms to end users.

scanSelectedText just scans the selected text, simple.

scanTextAtSelection scans the text starting **at** the selection and does whatever it wants inside or outside the selection.

Reply:
Oh yeah these are user facing strings 🤔